src/vm/assemblynative.cpp

```diff
@@ -1299,7 +1304,46 @@ INT_PTR QCALLTYPE AssemblyNative::InitializeAssemblyLoadContext(INT_PTR ptrManag
     {
         // Initialize a custom Assembly Load Context
         CLRPrivBinderAssemblyLoadContext *pBindContext = NULL;
+
+        // Create a new AssemblyLoaderAllocator for an AssemblyLoadContext
+        AssemblyLoaderAllocator* loaderAllocator = new AssemblyLoaderAllocator();
```

Comment:
We should need to do this for regular non-unloadable load context. It is unnecessary overhead for them.